### ISSUE TITLE
Auto-completing tags

### ISSUE THREAD

**Issue by gcluley** [firesidelabs/fireside-features#18]
When editing an episode's details, I would like the tag field to auto-suggest tags based on past tags.

So, for instance, if I have previously tagged an episode "videogame", I won't have to remember on the new episode if it should be "video game", "videogames", or "videogam".  It will helpfully suggest what I've entered in past based on the first x letters.  This is how tags work in the WordPress admin console, for instance.

TL;DR: When a user begins to type, it suggests tags that have matching text.

**Comment by Glark** [comment]
This would be very helpful.
